src/v4/http/index.ts

```diff
@@ -296,7 +299,7 @@ export class InteractionWithPluginRequest
     status: number,
     builder?: V4PluginResponseBuilderFunc
   ): V4InteractionWithPluginResponse {
-    this.interaction.withStatus(status);
+    this.interaction.withStatus(reify(status) as number);
```

Comment:
I suggest removing the `as number`. You could do this by adding a type parameter to `reify`.